tasks/joomla.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Create Joomla Directories
+  file:
+    path: "{{ item }}"
+    state: directory
+    # mode: 0755
+  with_items:
+    - "{{ docker_home }}/joomla"
+
+- name: Create MySQL Directorys for Joomla
+  file:
+    path: "{{ item }}"
+    state: directory
+    # mode: 0755
+  with_items:
+    - "{{ docker_home }}/JoomlaDB"
```

Comment:
Please can you combine this with the previous step - `with_items` allows you to provide a list.

Additionally, the DB and Joomla data should fall within the same subdirectory, ie:
```
{{ docker_home }}/joomla/data
{{ docker_home }}/joomla/data
```